api/src/Entity/Camp.php

```diff
@@ -98,7 +98,7 @@ class Camp extends BaseEntity implements BelongsToCampInterface {
      * Lists for collecting the required materials needed for carrying out the programme. Each collaborator
      * has a material list, and there may be more, such as shopping lists.
      *
-     * @ORM\OneToMany(targetEntity="MaterialList", mappedBy="camp", orphanRemoval=true)
+     * @ORM\OneToMany(targetEntity="MaterialList", mappedBy="camp", orphanRemoval=true, cascade={"persist"})
```

Comment:
This overlaps with #2335. But I assume it wouldn't create a merge conflict, as long as it's literally the same.